src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -113,6 +115,32 @@ function isPreview() {
     return $(document).find('.navbar').text().indexOf('Preview') !== -1;
 }
 
+function isModeration() {
+    return $('#moderationHintButton').length !== 0;
+}
+
+function bindModerationHintButton() {
+    if (!isModeration()) {
+        return;
+    }
+    
+    var expandText = '[Show more]';
+    var closeText = '[Show less]';
+    $('#moderationHintButton').text(expandText);
+    
+    $('#moderationHintButton').click(function(event) {
```

Comment:
Since we use `$('#moderationHintButton')` multiple times here, it is good to extract it into the variable `$moderationHintButton`